### ISSUE TITLE
Public cvterm pages exposing vocabularies to users

### ISSUE THREAD

**Issue by spficklin** [tripal/tripal#64]
Copied from Drupal issue queue: https://www.drupal.org/node/2499301

Andrew expressed a desire for public cvterm pages that could be linked to in views and listing. These would provide addition information to users and could include term hierarchies, etc.

NOTE: currently we have administrative pages only for cv/cvterms and even then we don't have a view (only edit) page.

**Comment by laceysanderson** [comment]
I would also like public cvterm pages. In a Tripal 3 context, I think we should be able to choose the cvterm table as the base for a Tripal Entity with filtering available based on cv_id. One use case for this is Andrew's wish for public pages describing his ontologies, and another is that in KnowPulse Phenotype Traits are stored as cvterms which we would want trait pages for. I don't think making Tripal Terms have public pages is sufficient because that really only reflects your content types, not all cvterms in an ontology.

**Comment by spficklin** [comment]
These pages were created when the web services were created because local terms needed a URL.  But the pages support all installed vocabularies.  I've just added a tree browser.  Here's an example:

http://demo.tripal.info/3.x/cv/lookup/SO

But, we still need an entry page for all controlled vocabularies (via the 'Controlled Vocabularies' breadcrumb.

I had to add a new materialized view to make it work:  db2cv_mview (that relates the controlled vocabularies to the databases they belong to)
